store/store.go

```diff
@@ -235,12 +236,26 @@ func NewStore(
 			}
 		})
 		if err != nil {
-			log.Fatalf("Cloud pubsub receive: %v", err)
+			log.Printf("Cloud pubsub receive: %v", err)
 		}
 	}()
 
 	// Initial branch cachel load.
-	err = st.LoadBranchCache(ctx, gcsFolder)
+	gcsClient, err := storage.NewClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+	rc, err := gcsClient.Bucket(gcsBucket).Object(versionFile).NewReader(ctx)
+	if err != nil {
+		log.Fatalf("%s", err)
```

Comment:
Are you sure we want to crash here (and below)? If so, crash in 246 too? (for consistency)

Reply:
Also made it Fatal. This is during server start, can crash here to make sure cache always loads